web/src/lib/utils/date.ts

```diff
@@ -1,5 +1,11 @@
 import { get } from "svelte/store";
 import { locale } from "./i8n";
+import { DateFormats } from "../constants/date";
+
+/**
+ * Date used to format times.
+ */
+const UNIX_TIME = "1970-01-01";
```

Comment:
Since this constant can be used in different scenarios, I would make the documentation more generic:
```suggestion
/**
 * Date the Unix time 0.
 */
const UNIX_TIME_0_DATE = "1970-01-01";
```